tests/scripts/thread-cert/command.py

```diff
@@ -305,10 +309,45 @@ def check_child_id_request(command_msg, tlv_request = CheckType.OPTIONAL, \
     check_tlv_request_tlv(command_msg, CheckType.CONTAIN, mle.TlvType.ADDRESS16)
     check_tlv_request_tlv(command_msg, CheckType.CONTAIN, mle.TlvType.NETWORK_DATA)
 
+def find_prefix_tlv(tlvs, cond_map):
+    """Find a prefix tlv in tlvs which matchs some conditions specified by cond_map
+    """
+    for tlv in tlvs:
+        if 'prefix' in cond_map:
+            if binascii.hexlify(tlv.prefix) != cond_map['prefix']:
+                continue
+        if 'br16' in cond_map:
+            border_router_tlv = get_sub_tlv(tlv.sub_tlvs, network_data.BorderRouter)
+            if border_router_tlv.border_router_16 != cond_map['br16']:
+                continue
+        return tlv
+    return None
+
+def check_network_data(data, check_detail):
+    check_type = check_detail[0]
+    prefixes = [tlv for tlv in data.tlvs if isinstance(tlv, network_data.Prefix)]
+    if check_type == NetworkDataCheckType.PREFIX_CNT:
+        # check_detail[1] should be a integer number representing the minimum cnt of prefixes should be
+        mn_cnt = check_detail[1]
+        assert len(prefixes) >= mn_cnt, 'Network data should contain at least {} prefixes'.format(mn_cnt)
+        for prefix in prefixes:
+            check_prefix(prefix)
+    elif check_type == NetworkDataCheckType.PREFIX_CONTENT:
+        # check_detail[1] should be a list of dictionary(like [{'prefix'='...', 'rloc16'='...'}])
```

Comment:
`rloc16` in the comment is not consistent with definitions in other places ‘br16'.
Additionally, how about also providing an enumeration for allowed keys to make it easier to use the interfaces without looking into the implementation details too much?

Reply:
Yeah, that's quite necessary. And I find that there are more and more enumerations used, I would organize them in later PRs maybe.